containers/mobilenet/dataset/scripts/tar_to_record.py

```diff
@@ -54,7 +54,16 @@ def main(dataset_paths, percent_eval, directory):
 
         #Unzip the zip in correct dir
         with zipfile.ZipFile(dataset_paths[-1], 'r') as zip_file: # Unzip the file (Assuming 1 zip at this time)
-            zip_file.extractall(EXTRACT_PATH)
+            print("test1 "+EXTRACT_PATH+"/"+DATASET_NAME)
+            zip_file.extractall(EXTRACT_PATH+"/"+DATASET_NAME)
```

Comment:
```suggestion
            zip_file.extractall(EXTRACT_PATH)
```
This change allows uno dataset to train, otherwise cant generate map.pbtxt